Source/WebCore/html/HTMLTemplateElement.cpp

```diff
@@ -105,6 +105,16 @@ void HTMLTemplateElement::setShadowRootMode(const AtomString& value)
     setAttribute(HTMLNames::shadowrootmodeAttr, value);
 }
 
+const AtomString& HTMLTemplateElement::shadowRootReferenceTarget() const
+{
+    return attributeWithoutSynchronization(HTMLNames::shadowrootreferencetargetAttr);
```

Comment:
Should this have a settings check?

Reply:
My gut reaction is no, since the attribute is settings-dependent:

```cpp
[EnabledBySetting=ShadowRootReferenceTargetEnabled] attribute [AtomString] DOMString referenceTarget;
```

But I'm not super familiar with how that works, so maybe a good question to carry forward to your review request.